src/selectors/profile.js

```diff
@@ -562,7 +563,7 @@ export const getHiddenTrackCount: Selector<HiddenTrackCount> = createSelector(
  */
 export const getPagesMap: Selector<Map<
   BrowsingContextID,
-  Set<InnerWindowID>
+  Page[]
```

Comment:
It was a `Set<InnerWindowID>` before because we needed to access its values randomly before (mostly for `has` checks during the sample/marker filtering). But now we don't need to do this for `getPagesMap` because we actually need a sequential access for active tab track calculation. For the Set part, I separated the previous `getPagesMap` into multiple pieces. Now newly created `getInnerWindowIDSetByBrowsingContextID` selector actually does that step and returns it.